lib/ansible/modules/source_control/github_release.py

```diff
@@ -201,15 +201,14 @@ def main():
     if action == 'create_release':
         release_exists = repository.release_from_tag(tag)
         if release_exists:
-            module.exit_json(
-                skipped=True, msg="Release for tag %s already exists." % tag)
+            module.exit_json(changed=False, msg="Release for tag %s already exists." % tag)
```

Comment:
Shouldn't skipped=True' be kept, for backward compatibility ?

Reply:
No. I will mention it in documentation about the changed state behavior.